dspy/utils/usage_tracker.py

```diff
@@ -40,6 +42,8 @@ def _merge_usage_entries(self, usage_entry1: dict[str, Any] | None, usage_entry2
             current_v = result.get(k)
             if isinstance(v, dict) or isinstance(current_v, dict):
                 result[k] = self._merge_usage_entries(current_v, v)
+            elif hasattr(v, "__dict__") or hasattr(current_v, "__dict__"):
```

Comment:
Do we need to expose `input_tokens_details` or `output_tokens_details` in dspy? It might be confusing if we use both chat completion and responses together. Can we just filter out these new fields for now?